one-app-statics/index.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+throw new Error('one-app-statics is not intended to be required, please refer to the README for more details');
```

Comment:
if they require these ... why do we mind?

Reply:
no apart from having bad DX. if they want to i think they could still require a static directly.

Reply:
Agree, this only applies to the npm package that hosts the files, I guess if they want to use the statics they can go to JSDelivr directly instead of npm